### PR TITLE
Associations

### PR DESCRIPTION
**Pull request by jadibdev**
### Description
Create the rental, book, and classroom classes. Created associations between the different classes.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -9,4 +9,9 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
   def play_hooky
     "¯\(ツ)/¯"
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
```
Comment (vechicin) [inline student.rb:19]: You need a getter for `classroom` in this class. Please add it.

File: book.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
```
Comment (vechicin) [inline book.rb]: You need a setter and a getter for `rentals` as well. Please add it.

File: rental.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class Rental
+  attr_accessor :date
```
Comment (vechicin) [inline rental.rb:5]: You need getters for `book` and `person`, please add them.

File: person.rb [hunk 4]
```diff
@@ -7,6 +7,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   attr_accessor :name, :age
```
Comment (vechicin) [inline person.rb]: You need a setter and getter for `rentals`. Please add them.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @jadibdev ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job implementing the associations necessary for this milestone

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @jadibdev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**